vmdb/app/models/miq_request_task.rb

```diff
@@ -62,6 +62,11 @@ def update_request_status
     # Determine status to report
     req_status = status.slice('Error', 'Timeout', 'Warn').keys.first || 'Ok'
 
+    if req_state == "finished" && state != "finished"
+      req_state = (req_status == 'Ok') ? 'provisioned' : "finished"
```

Comment:
@gmcculloug 
As you and I discussed, this code change only really affects service provisioning so "provisioned" is an acceptable post_execution state.  I don't believe the post_execution state of "migrated" or any other very specific post_execution state is necessary and will look into the possibility of standardizing that in a future release.